price/priceService.ts

```diff
@@ -1,9 +1,23 @@
 import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
 
-type Pair = string;
 type TimeFrame = string;
 
-async function fetchData(pair: Pair, timeFrame: TimeFrame): Promise<string> {
+interface candles {
+  startedAt: "string",
+  ticker: "string",
+  resolution: "1MIN",
+  low: "string",
+  high: "string",
+  open: "string",
+  close: "string",
+  baseTokenVolume: "string",
+  usdVolume: "string",
+  trades: "int",
```

Comment:
`"int"` is called a string literal type in Typescript. In essence, it means only a string with a value of `"int"`. I'm pretty sure you didn't mean it for `trades`.

Reply:
soooryyyyy🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️